addons/account/models/account_move.py

```diff
@@ -1340,6 +1341,11 @@ def _compute_needed_terms(self):
                         'amount_currency': invoice.amount_total_in_currency_signed,
                     }
 
+    def _compute_show_journal(self):
+        self.show_journal = False
```

Comment:
```suggestion
    def _compute_show_journal(self):
        for move in self:
            move.show_journal = False
```
also please update the entire method accordingly